packages/docprovider/src/yprovider.ts

```diff
@@ -21,15 +21,15 @@ import { IDocumentProvider, IDocumentProviderFactory } from './tokens';
  * We specify custom messages that the server can interpret. For reference please look in yjs_ws_server.
  *
  */
-export class WebSocketProviderWithLocks
-  extends WebsocketProvider
+export class WebSocketProvider
```

Comment:
This renaming and the removal of `acquireLock` and `releaseLock` from the interfaces make this PR not backward compatible anymore.

Reply:
It also makes the tests fail and I don't understand why: https://github.com/jupyterlab/jupyterlab/runs/5597604691?check_suite_focus=true.
Let's keep the room locks for now and remove them in another PR.